### ISSUE TITLE
build/test OSX on travis?

### ISSUE THREAD

**Issue by jrobeson** [beetbox/beets#2025]
I'm surprised this isn't already there, so maybe there's a good reason it isn't.

**Comment by sampsyo** [comment]
We never bothered to set it up because it's not likely to reveal any bugs that the Linux platform won't. (Unlike running on Windows, which is all kinds of different.) But it can't hurt!

**Comment by jrobeson** [comment]
It would shake any out any differences whenever we call external tooling that differs between OSX (and most BSDs) and ones that exclusively use the GNU variants (Linux).

(I know we don't call much external tooling in general, but I'm sure we'll be doing it for some things)

It would also make sure we never screw up case sensitivity on Unix, since HFS is mounted as case sensitive by default on OSX.

It's a pretty good proxy for making sure it works on most of the BSDs in general.

**Comment by sampsyo** [comment]
Yep, and it certainly doesn't cost us anything to add it! Please go ahead and add a travis.yml entry if you're interested.
